src/lib/components/error-pages/ErrorPage404.component.js

```diff
@@ -77,10 +77,10 @@ function ErrorPage404({ btnLink = '/', locale = 'en', ...rest }: Props) {
       </Row>
       <Row>
         <Description>
-          <DescriptionContent aria-label="page not found">
+          <DescriptionContent aria-labelledby="page-not-found">
```

Comment:
I'm not sure this is correct/ nor needed. 
According to MDN https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute  aria-labelledby attribute accepts a list of id of other dom element. So here the id tag is missing. Also as you mentioned as the text is readable and explicit there is no need of any extra aria property. Screen readers would just read the content of the element naturally

Reply:
You are right!
I'll remove the `aria-label` in the error pages